index.html

```diff
@@ -3,13 +3,14 @@
   <head>
     <meta charset="utf-8">
     <title>Startrly</title>
+    <link rel="stylesheet" type="text/css" media="screen" href="normalize.css">
+    <link rel="stylesheet" type="text/css" media="screen" href="style.css">
   </head>
   <body>
     <div>
 
-
       <nav>
-        <ul>
+        <ul class="nav">
```

Comment:
Did you need to add this class? Since there is only one nav element, this ul could be accessed in css by using the selector "nav ul {}"